config/manager/manager.yaml

```diff
@@ -28,7 +28,7 @@ spec:
         args:
         - --enable-leader-election
         image: controller:latest
-        imagePullPolicy: Always
+        imagePullPolicy: IfNotPresent
```

Comment:
As per https://kubernetes.io/docs/concepts/containers/images/#updating-images , you can force image pulls by leaving the default setting: `IfNotPresent` + using the tag `latest`. `Always` is not required.